src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -15,10 +15,22 @@
  */
 
 import React from 'react';
+import { useAppSelector } from 'src/store';
+import { useLocation } from 'react-router';
+
+import { pluralise } from 'src/shared/helpers/formatters/pluralisationFormatter';
 
 import BlastAppBar from 'src/content/app/tools/blast/components/blast-app-bar/BlastAppBar';
 import ToolsTopBar from 'src/content/app/tools/shared/components/tools-top-bar/ToolsTopBar';
 import BlastViewsNavigation from 'src/content/app/tools/blast/components/blast-views-navigation/BlastViewsNavigation';
+import { Header } from 'src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission';
```

Comment:
`ListedBlastSubmission` is a component is shown on the page containing all unviewed/viewed submissions. `BlastSubmissionResults` (not sure if it's a good name) is a different component, displayed on the page that focuses on a single submission.

We shouldn't be exporting the `Header` component from the `ListedBlastSubmission` component and importing it in `BlastSubmissionResults` — `BlastSubmissionResults` should not depend on `ListedBlastSubmission`. If we are sufficiently confident that this header is going to be the same across the two components, and that the only difference will be that it will either have a chevron (in the list view) or not (in the single submission view), then we should extract it into its own component. If, however, the headers are sufficiently different between the two components, then we should just recreate the header anew.